rules-placeholder/windows/network_connection/net_connection_win_rdp_from_dc.yml

```
@@ -0,0 +1,25 @@
+title: RDP Originating From Domain Controller
+id: fda34293-718e-4b36-b018-38caab0d1209
+status: experimental
+description: Detects a RDP connection originating from a domain controller
+author: Josh Nickels
+date: 2024/04/30
+references:
+    - internal research
+logsource:
+    product: windows
+    category: network_connection
+detection:
+    selection_port:
+        DestinationPort: 3389
+    selection_hosts:
+        ComputerName|expand: '%domain_controller_hostnames%'
```

```suggestion
        Computer|expand: '%domain_controller_hostnames%'
```
In the windows event it is `Computer`